capsules/classifier_safety_gear_openvino/capsule.py

```diff
@@ -0,0 +1,36 @@
+from vcap import BaseCapsule, NodeDescription, DeviceMapper
+
+from .backend import Backend
+
+from .config import (
+    safety_hat,
+    safety_vest,
+    with_safety_hat,
+    with_safety_vest,
+    capsule_options,
+)
+
+
+class Capsule(BaseCapsule):
+    name = "classifier_safety_gear_openvino"
+    description = "Roughly identify if person is wearing safety hat " \
+                  "and safety vest."
+    version = 1
+    device_mapper = DeviceMapper.map_to_openvino_devices()
+    input_type = NodeDescription(
+        size=NodeDescription.Size.ALL,
+        detections=["person"])
+    output_type = NodeDescription(
+        size=NodeDescription.Size.ALL,
+        detections=["person"],
+        attributes={safety_hat: with_safety_hat,
+                    safety_vest: with_safety_vest},
+        extra_data=["safety_hat_iou", "safety_hat_confidence",
+                    "safety_vest_iou", "safety_vest_confidence"],
+    )
+    backend_loader = lambda capsule_files, device: Backend(
```

Comment:
I just remembered- currently we are wrapping all OpenVINO backends with a BackendRPCProcess, due to instabilities with OpenVINO and the devkit. Could you do this:

```
from vcap_utils import BackendRPCProcess

BackendRPCProcess(
        Backend,
        model_xml=capsule_files["worker_safety_mobilenet_FP16.xml"],
        weights_bin=capsule_files["worker_safety_mobilenet_FP16.bin"],
        device_name=device
)
```

Reply:
Is this also a vcap-v0.3 feature? I'm getting errors when load into bf v0.27.

Reply:
This is a vcap-0.2.7 feature. There are other capsules on the 0.2 branch that use this: [example](https://github.com/aotuai/capsule_zoo/blob/17c63b1a4d35298aaa150f51b92e786751ad4df1/capsules/classifier_vehicle_color_openvino/capsule.py#L20)

Copied from the link
```python
    backend_loader = lambda capsule_files, device: BackendRpcProcess(
        Backend,
        model_xml=capsule_files["vehicle-attributes-recognition-barrier-0039.xml"],
        weights_bin=capsule_files["vehicle-attributes-recognition-barrier-0039.bin"],
        device_name=device
    )
```